src/ucp/wireup/address.c

```diff
@@ -787,7 +787,9 @@ static ucs_status_t ucp_address_do_pack(ucp_worker_h worker, ucp_ep_h ep,
             if (!(pack_flags & UCP_ADDRESS_PACK_FLAG_NO_TRACE)) {
                ucs_trace("pack addr[%d] : "UCT_TL_RESOURCE_DESC_FMT" "
                           "eps %u md_flags 0x%"PRIx64" tl_flags 0x%"PRIx64" bw %e + %e/n ovh %e "
-                          "lat_ovh %e dev_priority %d a32 0x%lx/0x%lx a64 0x%lx/0x%lx",
+                          "lat_ovh %e dev_priority %d a32 "
+                          "0x%"PRIx64"/0x%"PRIx64" a64 "
+                          "0x%"PRIx64"/0x%"PRIx64,
```

Comment:
wrap lines

Reply:
Could you check again? I wrapped by 80 characters.